start/scaling-up/preparing-to-scale-up.md

```diff
@@ -132,12 +152,35 @@ desired output, and do not conflict with each other when submitted together. Onc
 you are confident that the jobs will complete as desired, then scale up to submitting 
 the entire set of jobs.
 
-## Avoid Exceeding Disk Quotas
+# Monitoring Job Status and Obtaining Run Information
+
+Gathering information about how, what, and where a job ran can be important for both troubleshooting and optimizing a workflow. The following commands are a great way to learn more about your jobs: 
+
+| Command      | Description |
+| ----------- | ----------- |
+| `condor_q`      | Shows the queue information for your jobs. Includes information such as batch name and total jobs. |
+| `condor_q <JobID> -l` | Prints all information related to a job including attributes and run information about a job in the queue. Output includes `JobDurationCategory`, `ServerTime`, `SubmitFile`, etc. |
+| `condor_q <JobID> -af <AttributeName1> <AttributeName2>` | Prints information about an attribute or list of attributes for a single job using the autoformat `-af` flag. The list of possible attributes can be found using `condor_q <JobID> -l`. |
+| `condor_q -constraint '<Attribute> == "<value>"'   | The `-constraint` flag allows users to find all jobs with a certain value for a given parameter. This flag supports searching by more than one parameter and different operators (e.g. `=!=`).  |
+| condor_q -better-analyze <JobID> -pool <PoolName> | Shows a list of the number of slots matching a job's requirements. For more information, see [Troubleshooting Job Errors] (https://support.opensciencegrid.org/support/solutions/articles/5000639785-troubleshooting-job-errors).|
+
+Additional `condor_q` flags involved in optimizing and troubleshooting jobs include:
+| Flag      | Description |
+| ----------- | ----------- |
+| -nobatch | Combined with `condor_q`, this flag will list jobs individually and not by batch. |
+| -all | Shows all jobs submitted on the submit server.|

```

Comment:
I don't know if we need this option for this particular guide - usually you only care about your jobs.

Reply:
I agree, I removed it.